test/conformance/feature/conformance_feature.go

```diff
@@ -0,0 +1,53 @@
+package feature
```

Comment:
should we do some headers? 

and do we need instructions, like:

```
// +build e2e

```

here too?

Reply:
We don't need e2e build tags I think (at least, i don't see them in eventing).

I added the copyright headers